pages/common/jello.md

```diff
@@ -0,0 +1,32 @@
+# jello
+
+> A command-line JSON processor using python syntax.
+> More information: <https://github.com/kellyjonbrazil/jello>.
+
+- Output JSON or JSON Lines STDIN data, in pretty-print format:
+
+`cat {{file.json}} | jello`
+
+- Output a grepable schema from JSON or JSON Lines STDIN data:
+
+`cat {{file.json}} | jello -s`
+
+- Output all elements from arrays (or all the values from objects) in JSON or JSON Lines STDIN data:
+
+`cat {{file.json}} | jello -l`
+
+- Output the first element in JSON or JSON Lines STDIN data:
+
+`cat {{file.json}} | jello _[0]`
+
+- Output the value of a given key of each element in JSON or JSON Lines STDIN data:
+
+`cat {{file.json}} | jello '[i.{{key_name}} for i in _]'`
+
+- Output the value of multiple keys as a new JSON object (assuming the input JSON has the keys `key_name` and `other_key_name`):
+
+`cat {{file.json}} | jello '{{{"my_new_key"}}: _.{{key_name}}, {{"my_other_key"}}: _.{{other_key_name}}}'`
```

Comment:
```suggestion
`cat {{file.json}} | jello '"{{my_new_key}}": _.{{key_name}}, "{{my_other_key}}": _.{{other_key_name}}}'`
```
We don't really count the `" "` to be part of the argument here.

Reply:
Ah, that makes sense. I still need that extra `{` in the very beginning, though.